### DISCUSSION TITLE
地图导入问题

### DISCUSSION THREAD

**Discussion by Chenyantt** [liusy58/NJU_AdvancedProgramming_SP22#196] [Q&A]
请问地图的导入是指我要在前端实现一个用户可以导入文件的接口，还是指只要在后台允许用户将符合格式的txt文件拷贝到相应的文件里，并根据txt内容生成对应地图即可。

**Comment by liusy58** [comment]
后者

**Reply by myalice-forever** [reply]
请问地图导出是必须要实现的内容吗，如果是的话，请问怎么才算“合格”的导出

**Reply by liusy58** [reply]
印象中地图的导入导出是选做内容
